### PR TITLE
[10] Create & Style the Customer Signup Page

### PR DESCRIPTION
**Pull request by MYKEL7**
## Task: `[10] Create & Style the Customer Signup Page`
- `I created and styled the SignUp page`

bigger layout

Tablet layout

mobile layout

### CODE REVIEW

**Comment by adaugochi** [comment]
I can't find your css file

**Comment by MYKEL7** [comment]
Done @adaugochi

**Comment by MYKEL7** [comment]
Done.

File: submissions/onyemeMichael/css/signUp.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (adaugochi) [inline submissions/onyemeMichael/css/signUp.html:2]: Please place your signup form inside your HTMLforms folder

File: submissions/charlesBanda/html/index.html [hunk 2]
```diff
@@ -1,81 +1,175 @@
...
+					<th><a href="#">Register</a></th>
+					<th><a href="#">What's Hot!</a></th>
+				</tr>
+			</table>
+		</header>
+		<div>
+			<table>
+				<tr>
+					<th>
+		          <h1><font face="Arial">Msika</font></h1>
```
Comment (ToluAkin) [inline submissions/charlesBanda/html/index.html:23]: @MYKEL7 Remove this `<font face="Arial">` you can set font-family in your css.

File: submissions/charlesBanda/html/index.html [hunk 3]
```diff
@@ -1,81 +1,175 @@
...
+			<ul>
+					<li><a href="http://www.facebook.com" target="_blank">Facebook</a></li>
+					<li><a href="http://www.twitter.com" target="_blank">Twitter</a></li>
+					<li><a href="http://www.pinprest.com" target="_blank">Pinprest</a></li>
+			</ul>
+</body>
+<!--copyright-->
+<footer><p>&copy; 2020 Msika.com<p></footer>
+</html>
+=======
```
Comment (ToluAkin) [inline submissions/charlesBanda/html/index.html:94]: @MYKEL7 Remove this line

File: submissions/charlesBanda/html/index.html [hunk 4]
```diff
@@ -1,81 +1,175 @@
...
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/charlesBanda/html/index.html:95]: Why are you adding another Html tag here. You should move it into another file or remove it completely

**Comment by MYKEL7** [comment]
Done.

File: submissions/charlesBanda/html/index.html (hunk 2 above)
Comment (MYKEL7) [reply submissions/charlesBanda/html/index.html:23]: I dont know any thing about this

File: submissions/charlesBanda/html/index.html (hunk 3 above)
Comment (MYKEL7) [reply submissions/charlesBanda/html/index.html:94]: I don't know any thing about this.

File: submissions/charlesBanda/html/index.html (hunk 2 above)
Comment (adaugochi) [reply submissions/charlesBanda/html/index.html:23]: Remove it. Then style the font in your css instead

File: submissions/ejikeIweriebor/html/product_page.html [hunk 5]
```diff
@@ -1,104 +1,104 @@
-<!DOCTYPE html>
-<html lang="en">
-
```
Comment (adaugochi) [inline submissions/ejikeIweriebor/html/product_page.html:3]: Does this file belong to you? I think this is someone else's file.

File: submissions/onyemeMichael/css/signUp.css [hunk 6]
```diff
@@ -0,0 +1,226 @@
+body {
+    font-family: 'Arimo', sans-serif;
+    font-size: 15px;
```
Comment (adaugochi) [inline submissions/onyemeMichael/css/signUp.css:3]: This file belongs to someone else

**Comment by adaugochi** [comment]
You pushed people files along with yours

**Pull request closed without merging**